libs/alcotest_ext/TODO.md

```diff
@@ -7,7 +7,8 @@ its own repository.
 Things to do:
 
 * rename Alcotest_ext to something shorter
-* show full test output as part of the "full status" output
 * make color output optional (on/off/auto)
 * protect against tests that might change terminal settings mid-run
   and cause color to disappear mid-run (already an issue with plain Alcotest)
+* use the Duff library for portable, color diff ('diff --color' is not
```

Comment:
might be good to add a small README in the library explaining quickly why we defined yet another testing library, what's the main differences with ounit and alcotest.